testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7534,6 +7534,10 @@ private static void checkArrayConcatAggFuncFails(SqlOperatorFixture t) {
     f.checkScalar("timestampdiff(\"month4\", date '2016-02-24', "
             + "date '2016-02-23')",
         "0", "INTEGER NOT NULL");
+    f.withLibrary(SqlLibrary.BIG_QUERY).setFor(SqlLibraryOperators.TIMESTAMP_DIFF)
+        .checkScalar("timestamp_diff(timestamp '2008-12-25 15:30:00', "
+            + "timestamp '2008-12-25 16:30:00', \"minute15\")",
+            "4", "INTEGER NOT NULL");
```

Comment:
My understanding was that since timestamp_diff just uses a timeunit as opposed to an interval like timestamp_add, it should be able to support custom timeframes. Should I put this test in the timestamp_diff-specific test instead?

Reply:
My mistake. This test is fine here. 

I’d break the line before “.setFor”. Long lines are difficult to maintain.